plugins/volumeFilter_test.go

```diff
@@ -0,0 +1,439 @@
+			wantOp:    nil,
+			wantError: nil,
+			wantTBB:   &VolumeFilterConfig{SellBaseAssetCapInBaseUnits: createFloatPtr(0.0), SellBaseAssetCapInQuoteUnits: createFloatPtr(0.0)},
+		},
+		{
+			name: "success - selling, quote units sell cap, keep selling quote",
+			filter: &volumeFilter{
+				name:                   "volumeFilter",
+				configValue:            "",
+				baseAsset:              testNativeAsset,
+				quoteAsset:             testNativeAsset,
+				config:                 &VolumeFilterConfig{SellBaseAssetCapInQuoteUnits: createFloatPtr(1.0), mode: volumeFilterModeExact},
+				dailyVolumeByDateQuery: dailyVolumeByDateQuery,
+			},
+			dailyOTB: &VolumeFilterConfig{
+				SellBaseAssetCapInBaseUnits:  createFloatPtr(0.0),
+				SellBaseAssetCapInQuoteUnits: createFloatPtr(0.0),
+			},
+			dailyTBB: &VolumeFilterConfig{
+				SellBaseAssetCapInBaseUnits:  createFloatPtr(0.0),
+				SellBaseAssetCapInQuoteUnits: createFloatPtr(0.0),
+			},
+			op:        &txnbuild.ManageSellOffer{Buying: txnbuild.NativeAsset{}, Selling: txnbuild.NativeAsset{}, Price: "1.0", Amount: "100.0"},
+			wantOp:    &txnbuild.ManageSellOffer{Buying: txnbuild.NativeAsset{}, Selling: txnbuild.NativeAsset{}, Price: "1.0", Amount: "1.0000000"},
+			wantError: nil,
+			wantTBB:   &VolumeFilterConfig{SellBaseAssetCapInBaseUnits: createFloatPtr(1.0), SellBaseAssetCapInQuoteUnits: createFloatPtr(1.0)},
+		},
+		{
+			name: "success - selling, base and quote units sell cap, keep selling base and quote",
+			filter: &volumeFilter{
+				name:                   "volumeFilter",
+				configValue:            "",
+				baseAsset:              testNativeAsset,
+				quoteAsset:             testNativeAsset,
+				config:                 &VolumeFilterConfig{SellBaseAssetCapInBaseUnits: createFloatPtr(1.0), SellBaseAssetCapInQuoteUnits: createFloatPtr(1.0), mode: volumeFilterModeExact},
+				dailyVolumeByDateQuery: dailyVolumeByDateQuery,
+			},
+			dailyOTB: &VolumeFilterConfig{
+				SellBaseAssetCapInBaseUnits:  createFloatPtr(0.0),
+				SellBaseAssetCapInQuoteUnits: createFloatPtr(0.0),
+			},
+			dailyTBB: &VolumeFilterConfig{
+				SellBaseAssetCapInBaseUnits:  createFloatPtr(0.0),
+				SellBaseAssetCapInQuoteUnits: createFloatPtr(0.0),
+			},
+			op:        &txnbuild.ManageSellOffer{Buying: txnbuild.NativeAsset{}, Selling: txnbuild.NativeAsset{}, Price: "1.0", Amount: "100.0"},
+			wantOp:    &txnbuild.ManageSellOffer{Buying: txnbuild.NativeAsset{}, Selling: txnbuild.NativeAsset{}, Price: "1.0", Amount: "1.0000000"},
+			wantError: nil,
+			wantTBB:   &VolumeFilterConfig{SellBaseAssetCapInBaseUnits: createFloatPtr(1.0), SellBaseAssetCapInQuoteUnits: createFloatPtr(1.0)},
+		},
+	}
+
+	for _, k := range testCases {
+		t.Run(k.name, func(t *testing.T) {
+			tbb := k.dailyTBB
+			actual, e := k.filter.volumeFilterFn(k.dailyOTB, tbb, k.op)
+			assert.Equal(t, k.wantError, e)
+			assert.Equal(t, k.wantOp, actual)
+			assert.Equal(t, k.wantTBB, tbb)
```

Comment:
it's confusing that tbb is modified in volumeFilterFn -- that's my fault.
Can you please rename it in the function and here to `dailyTBBAccumulator` -- the accumulator denotes that it's a modified variable, which helps distinguish it from OTB which is not modified